app/services/big_blue_button_api.rb

```diff
@@ -12,13 +12,57 @@ def bbb_server
     @bbb_server ||= BigBlueButton::BigBlueButtonApi.new(bbb_endpoint, bbb_secret, '1.8')
   end
 
+  # Start a session for a specific room and returns the join URL.
+  def start_session(room:, session_starter:, options: {})
+    # TODO: amir - Revisit this.
+    create_options = default_create_opts.merge(options)
+    join_options = default_join_opts.merge(create_options) # TODO: amir - Revisit this (createTime,...).
+
+    meeting_id = room.friendly_id
```

Comment:
I'd avoid this too - adds confusion around naming. Just set `meeting_id: room.friendly_id`

Reply:
Alright, I'll change it

Reply:
DONE